watertap/property_models/activated_sludge/asm1_properties.py

```diff
@@ -273,13 +276,21 @@ def build(self):
             self.params.solute_set,
             domain=pyo.NonNegativeReals,
             initialize=0.1,
-            doc="Component molar concentrations",
+            doc="Component mass concentrations",
             units=pyo.units.kg / pyo.units.m**3,
         )
+        self.alkalinity = pyo.Var(
+            domain=pyo.NonNegativeReals,
+            initialize=1,
+            doc="Alkalinity in molar concentration",
+            units=pyo.units.kmol / pyo.units.m**3,
+        )
 
     def get_material_flow_terms(b, p, j):
         if j == "H2O":
             return b.flow_vol * b.params.dens_mass
+        elif j == "alkalinity":
+            return b.flow_vol * b.alkalinity * (14 * pyo.units.kg / pyo.units.kmol)
```

Comment:
what is the 14 based on?

Reply:
I assume it is the molecular weight of N based on usage, but the source is not clear on that.

Reply:
Can we double-check on this? Alkalinity is usually on mmol HCO3-/L, we sometimes multiply it by 12 to convert it to mg C/L, so 14 representing the MW of N is a bit weird

Reply:
14 is the correct value, although I cannot say why thy used it. My guess is that it is because they are interested in alkalinity in terms of nitrogen species.

Reply:
OK as in #509 , so if this is for charge balance with NO3 and NO3 is expressed in kg N/m3, then it makes sense, thanks!